daml-lf/engine/src/main/scala/com/digitalasset/daml/lf/engine/Error.scala

```diff
@@ -89,6 +98,28 @@ object Error {
           case _ => None
         }
     }
+
+    final case class Type(
```

Comment:
not super clear what `Type` means. I assume you mean `TypeMismatch` or something like that

Reply:
:+1: